resources/assets/components/Post/test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import Post from './index';
+
+test('it renders correctly', () => {
+  const post = {
+    caption: 'Here is my awesome caption!',
+  };
+
+  const signup = {
+    signup_id: 7,
+  }
+
+  const tree = renderer.create(
+    <Post post={post} signup={signup}></Post>
+  ).toJSON();
+  expect(tree).toMatchSnapshot();
```

Comment:
This looks great to me, @katiecrane ! @DFurnes just out of curiosity, is there much difference between using `react-test-renderer` vs. `shallow` from `enzyme`?

Reply:
Mmm, good question! Plain ol' `react-test-renderer` renders the full tree (including any child components), whereas `react-test-renderer/shallow` would work more like Enyzme's `shallow` helper.

I think Enyzme just gives some extra helpers on top of `react-test-renderer/shallow` to make it easier to make assertions on the rendered component, so it's probably not a big deal here (since we're just doing a snapshot test).

Reply:
ah that is good to know! thanks so much, @DFurnes !

Reply:
most interesting!!